File: OC_TencentVideoChannel/OC_TencentVideoChannel/Views/ORMyChannelHeader.m
//
//  ORMyChannelHeader.m
//  OC_TencentVideoChannel
//
//  Created by OrangesAL on 2018/7/18.
//  Copyright © 2018年 欧阳荣. All rights reserved.
//

#import "ORMyChannelHeader.h"

@implementation ORMyChannelHeader

- (void)awakeFromNib {
    [super awakeFromNib];
    
    self.sortSwitch.transform = CGAffineTransformMakeScale(0.7, 0.7);
}

- (IBAction)action_sortSwitch:(UISwitch *)sender {
    if (self.switchChanged) {
        self.switchChanged(sender.isOn);
    }
}


@end

File: Swift_TencentVideoChannel/TencentVideoChannel/Views/ORMyChannelHeader.swift
//
//  ORMyChannelHeader.swift
//  TencentVideoChannel
//
//  Created by 欧阳荣 on 2018/7/17.
//  Copyright © 2018 欧阳荣. All rights reserved.
//

import UIKit

class ORMyChannelHeader: UICollectionReusableView {
    @IBOutlet weak var sortDescriptionL: UILabel!
    @IBOutlet weak var sortSwitch: UISwitch!
    
    var switchChanged:((Bool)->Void)?
    
    
    @IBAction func action_sortSwitch(_ sender: UISwitch) {
        if switchChanged != nil {
            switchChanged!(sender.isOn)
        }
    }
    
    override func awakeFromNib() {
        super.awakeFromNib()
        self.sortSwitch.transform = CGAffineTransform.init(scaleX: 0.7, y: 0.7)
    }
}
